jp/components/toc.yml

```diff
@@ -19,32 +19,32 @@
     - name: プロジェクト テンプレートの認証
       href: general/cli/auth-template.md
       new: false
-- name: WPF to Angular guide
+- name: WPF vs Angular 比較ガイド
```

Comment:
@yoshimis All the topics bellow are in English. We should hide them if they won't be localized before the next JP push to production.